3rdparty/libunwind/stubs.h

```diff
@@ -90,7 +90,7 @@ static __inline void* __libunwind_mmap(
     int fd,
     off_t offset)
 {
-    void* dlmemalign(size_t alignment, size_t size);
+    void* _oe_memalign(size_t alignment, size_t size);
```

Comment:
The underscore prefix i.e. _ should be dropped according to the OE coding standards as the single underscore prefix is reserved for local definitions. Please change this to oe_memalign.

Reply:
Can someone from the OE team please provide guidance here? The headers in `include/openenclave/internal` use a single prefixed underscore in some places (structs), a double prefixed underscore in others (files.h, globals.h, etc), but the preference seems to be for no descriminator for internal but wider than module-scope functions. We're happy to use any name for these that suits the OE team, of course!

Reply:
I'm sorry for the inconsistencies. Our coding convention is to use single underscores to introduce static function names. We didn't do a good job of publicizing this and some of our team members missed it. Of the options above, I liked "oe_internal_malloc()".